test/e2e/framework/util.go

```diff
@@ -2421,20 +2421,40 @@ func WaitForAllNodesSchedulable(c clientset.Interface, timeout time.Duration) er
 	})
 }
 
-func GetTTLAnnotationFromNode(node *v1.Node) (time.Duration, bool) {
+func GetPodSecretUpdateTimeout(c clientset.Interface) time.Duration {
+	// With SecretManager(ConfigMapManager), we may have to wait up to full sync period +
+	// TTL of secret(configmap) to elapse before the Kubelet projects the update into the
+	// volume and the container picks it up.
+	// So this timeout is based on default Kubelet sync period (1 minute) + maximum TTL for
+	// secret(configmap) that's based on cluster size + additional time as a fudge factor.
+	secretTTL, err := GetNodeTTLAnnotationValue(c)
+	if err != nil {
+		Logf("Couldn't get node TTL annotation: %v", err)
```

Comment:
Sorry for being boring, but can you also please explicitly log that we will be using "0" as a default value here?